site/layouts/partials/docs-versions.html

```diff
@@ -11,10 +11,10 @@
 {{- end }}
 
 <li class="nav-item dropdown">
-  <button class="btn btn-link nav-link py-2 px-0 px-lg-2 dropdown-toggle" id="bd-versions" data-bs-toggle="dropdown" aria-expanded="false" data-bs-display="static">
-    <span class="d-lg-none">Bootstrap</span> v{{ .Site.Params.docs_version }}
+  <button class="btn btn-link nav-link py-2 px-0 px-lg-2 dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false" data-bs-display="static">
+    <span class="d-lg-none" aria-hidden="true">Bootstrap</span><span class="visually-hidden">Bootstrap&nbsp;</span> v{{ .Site.Params.docs_version }} <span class="visually-hidden">(switch to other versions)</span>
```

Comment:
Just a question here. Why do we need a `&nbsp;` in the `.visually-hidden` `<span>`?

Reply:
@patrickhlauke I'm also wondering, couldn't we just use a `nbsp;` and removethe newly added `<span class="visually-hidden">Bootstrap&nbsp;</span>`? Something like this:

```html
<span class="d-lg-none">Bootstrap</span>&nbsp;v{{ .Site.Params.docs_version }} <span class="visually-hidden">(switch to other versions)</span>
```

Reply:
no because due to the `d-lg-none` the "Bootstrap" text is completely removed (even from assistive technologies), so the accessible name on large screen would not include it.

whether we prefer the `&nbsp;` inside or outside of the `<span>` is probably a question of taste